inc/Nav_Menus/Component.php

```diff
@@ -122,7 +122,7 @@ public function is_primary_nav_menu_active() : bool {
 	 */
 	public function display_primary_nav_menu( array $args = [] ) {
 		if ( ! isset( $args['container'] ) ) {
-			$args['container'] = 'ul';
+			$args['container'] = false;
```

Comment:
From the documentation, [`$args['container']` should be a string](https://core.trac.wordpress.org/browser/trunk/src/wp-includes/nav-menu-template.php#L28). When a container is not needed, an empty string should be passed. @felixarntz Let me know if I missed something.